src/SFA.DAS.ApplyService.InternalApi/Startup.cs

```diff
@@ -215,7 +175,56 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env)
                     template: "{controller=Home}/{action=Index}/{id?}");
             });
         }
-        
+
+        private void ConfigHttpClients(IServiceCollection services)
+        {
+            var acceptHeaderName = "Accept";
+            var acceptHeaderValue = "application/json";
+            var handlerLifeTime = TimeSpan.FromMinutes(5);
+
+            services.AddHttpClient<AssessorServiceApiClient, AssessorServiceApiClient>(config =>
```

Comment:
Some of these clients are EPAO-related, are they needed in Apply now it's RoATP-centric?

Reply:
It should be used for RoEPAO checks only for gateway 😺